apps/system/fxa/test/marionette/lib/fxa.js

```diff
@@ -149,9 +151,19 @@ FxA.prototype = {
     },
     clickFxaLogin: function() {
       var element = this.client.findElement(FxA.Selectors.fxaLogin);
+      console.log('found element', element);
       this.client.helper
           .waitForElement(element)
           .tap();
+      console.log('tapped');
+    },
+    clickFxsyncLogin: function() {
+      var element = this.client.findElement(FxA.Selectors.fxsyncLogin);
+      console.log('found element', element);
+      this.client.helper
+          .waitForElement(element)
+          .tap();
+      console.log('tapped');
```

Comment:
not sure what's going on, but did you try book-ending the get-started click with a few sleeps to see if there's a timing issue not being accounted for by waitForElement?  marionette often leaves me scratching my head.